Java Grep README.md

vassairm opened the pull request.

vassairm committed:
Java Grep README.md

andrew-laniak-fuoco reviewed (changes requested):
Your md formatting seems to be off as it shows me plain text.  Try using the in-browser editor below to make sure the formatting is correct.

https://stackedit.io/

vassairm committed:
README formatting fixed

Wrote the Readme in IntelliJ and the formatting looked fine there. Needed to add spaces after my Heading #'s
